.gitignore

```diff
@@ -20,3 +17,8 @@ vendor/build/
 vendor/install/
 .DS_Store
 
+doc/source/gallery.rst
+doc/api/
+doc/html/
+doc/jekyll_tools/Gemfile.lock
```

Comment:
`Gemfile.lock` should be committed to the repository; it serves as a snapshot of the exact versions of the gems that you used so I can install the _same_ versions, not only an approximately matching set based on the version constraints in the `Gemfile`. It helps a lot in avoiding the dreaded "works for me" syndrome.

Reply:
yep, done